src/app/domain/backup/helpers/error.ts

```diff
@@ -0,0 +1,18 @@
+export class BackupError extends Error {
+  statusCode: number | undefined
+
+  constructor(message: string, statusCode: number | undefined) {
+    const stringifiedMessage = JSON.stringify({
+      message,
+      statusCode
+    })
+
+    super(stringifiedMessage)
+    this.name = 'BackupError'
+    this.statusCode = statusCode
+  }
+}
+
+export const STACK_ERRORS_INTERRUPTING_BACKUP = [
+  413 // quota exceeded
```

Comment:
This is not true all the time. You can have a 413 error when the file is too big (5 GB) for our storage service (swift). 

If you want to handle that correctly, see desktop: 

https://github.com/cozy-labs/cozy-desktop/blob/eb1d431d382b9e7dcb93c49421e0555c60085951/core/remote/errors.js#L274-L287

Reply:
can be done later, not a blocker ;)

Reply:
Did not know about this! I will update with this new PR : https://github.com/cozy/cozy-stack/pull/4069/files